docs/REST.md

```diff
@@ -187,10 +187,10 @@ Content-Type: application/json
         "manufacturer": "Live",
         "vendor": "Mouser",
         "location": "Shelf 14",
-        "isFaulty": false,
-        "faultDescription": "",
+        "allowCheckout": true,
         "price": 10.50,
-        "count": 20
+        "count": 20,
+        "inStock": 20
     }
 }
```

Comment:
The checkout endpoint should be changed to equipmentAddresses instead of items.  I will do this.